src/main/target/common_pre.h

```diff
@@ -83,6 +84,7 @@
 #define USE_PERSISTENT_MSC_RTC
 #define USE_MCO
 #define USE_DMA_SPEC
+#define USE_VTX_TABLE
```

Comment:
Is there a reason that we are adding this by MCU type, and not by flash size? This should not be MCU type dependent really.